runtime/doc/lua.txt

```diff
@@ -698,25 +698,26 @@ VIM.HIGHLIGHT                  				*lua-highlight*
 Nvim includes a function for highlighting a selection on yank (see for example
 https://github.com/machakann/vim-highlightedyank). To enable it, add
 >
- au TextYankPost * silent! lua require'vim.highlight'.on_yank()
+ au TextYankPost * silent! lua vim.highlight.on_yank()
 <
 to your `init.vim`. You can customize the highlight group and the duration of
 the highlight via
 >
- au TextYankPost * silent! lua require'vim.highlight'.on_yank("IncSearch", 500)
+ au TextYankPost * silent! lua vim.highlight.on_yank {higroup="IncSearch", timeout=250}
```

Comment:
Thoughts on defaulting this timeout to 150 ms? The default of 500 ms is quite long and feels clunky to me.

Love this feature btw!

Reply:
Ah, just noticed the PR already reduced it from 500 to 250. I'm good with 250.

Reply:
I do feel 150 is better, it feels really snappy to me whereas 250 still sorta feels clunky.

Reply:
I have no preference -- @clason what do you think?

Reply:
I thought 250 was a reasonable compromise. Let's make a brief poll:

* 👍 500ms
* 😄 250ms
* ❤️ 200ms (split the difference)
* 👎 150ms

Reply:
K, do 150 and then let's ship it.

Reply:
Should have picked a friendlier emoji for the shortest option :)